site/profile/manifests/ncsa/allow_ssh_from_bastion.pp

```diff
@@ -0,0 +1,45 @@
+# Allow incoming ssh from any bastion nodes
+#
+# @summary Allow incoming ssh from any bastion nodes
+#
+# @param
+#   bastion_nodelist
+#   Type: Array
+#   Desc: one or more hostnames / IPs / CIDRs
+#         Note: must contain at least 1 item
+# @param
+#   allow_groups
+#   Type: Array
+#   Desc: one or more LDAP / UNIX groups that are allowed to login from
+#         any of the nodes in bastion_nodelist
+#         Note: can be empty
+# @example
+#   include profile::allow_ssh_from_bastion
+#
+class profile::ncsa::allow_ssh_from_bastion (
+    Array[ String ]    $allow_groups,
+    Array[ String, 1 ] $bastion_nodelist,
```

Comment:
Is `1` allowed here?

Reply:
1 indicates minimum size ... ie: array cannot be empty